index.html

```diff
@@ -1,52 +1,3336 @@
+                id="q149"
+                data-name="Group"
+                class="q149"
+              >
+                <div class="outer_q150">
+                <div
+                  id="q150"
+                  data-name="Group"
+                  class="q150"
+                >
+                  <div class="outer_q151">
+                  <img
+                    id="q151"
+                    data-name="Vector"
+                    src="images/Vector_5.png"
+                    alt="Vector"
+                    class="q151"
+                  >
+                  </div>
+                </div>
+                </div>
+              </div>
+              </div>
+              <div class="outer_q152">
+              <div
+                id="q152"
+                data-name="Group"
+                class="q152"
+              >
+                <div class="outer_q153">
+                <div
+                  id="q153"
+                  data-name="Group"
+                  class="q153"
+                >
+                  <div class="outer_q154">
+                  <img
+                    id="q154"
+                    data-name="Vector"
+                    src="images/Vector.png"
+                    alt="Vector"
+                    class="q154"
+                  >
+                  </div>
+                </div>
+                </div>
+              </div>
+              </div>
+              <div class="outer_q155">
+              <div
+                id="q155"
+                data-name="Group"
+                class="q155"
+              >
+                <div class="outer_q156">
+                <div
+                  id="q156"
+                  data-name="Group"
+                  class="q156"
+                >
+                  <div class="outer_q157">
+                  <img
+                    id="q157"
+                    data-name="Vector"
+                    src="images/Vector_3.png"
+                    alt="Vector"
+                    class="q157"
+                  >
+                  </div>
+                </div>
+                </div>
+              </div>
+              </div>
+            </div>
+            </div>
+          </div>
+          </div>
+        </div>
+        </div>
+        <div class="outer_q158">
+        <div
+          id="q158"
+          data-name="Group 120"
+          class="q158"
+        >
+          <div class="outer_q159">
+          <img
+            id="q159"
+            data-name="image_geometry_3"
+            src="images/image_geometry_3.png"
+            alt="image_geometry_3"
+            class="q159"
+          >
+          </div>
+          <div class="outer_q160">
+          <div
+            id="q160"
+            data-name="Rectangle 50"
+            class="q160"
+          >
+          </div>
+          </div>
+          <div class="outer_q161">
+          <div
+            id="q161"
+            data-name="Get started"
+            class="q161"
```

Comment:
- [ ] The container of the section is a section tag. Kindly fix this.
- [ ] The button (e.g., "Get My CV") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design), kindly make see this project as a link or button with a hover color and pressed.
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li
- [ ] The technology in each category(e.g., "JavaScript", "HTML" and "CSS") are also a list ul and each of them a li.

### Please follow the Figma. and fix those issues
- [ ] Original

![pro](https://user-images.githubusercontent.com/55635977/125859925-81fa0f98-e6e0-4d20-b857-319969f412c5.png)

- [ ] Yours
![yo](https://user-images.githubusercontent.com/55635977/125859945-ea868014-d45f-4f5c-af60-4896fcef9e23.png)